msi/release.sh

```diff
@@ -140,6 +140,7 @@ SCRIPT_WINPATH="$(cd "$SCRIPT_PATH" && pwd -W | tr / \\\\)"
 BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
 (cat <<EOF
 <?xml version="1.0" encoding="utf-8"?>
+<?define SixtyFourBit$(test -z "${BITNESS#32}" || echo =${BITNESS#32}) ?>
```

Comment:
I *think* this property is needed in commits that come earlier in the series. If that is true, the definition needs to move earlier, too...